debug/test/browser/devtools.test.js

```diff
@@ -137,7 +137,11 @@ function getRoot(element) {
 	return element._children;
 }
 
-describe('devtools', () => {
+// Devtools are only supported in Chrome and Firefox anyway
+const supported = /Chrome|Firefox/.test(navigator.userAgent) && !/Edge/.test(navigator.userAgent);
```

Comment:
Not sure how picky should we be here 😄 but this returned `true` for a mobile userAgent. We could iterate over it and maybe combine them rules? 
```js
/(Chrome)|(Firefox)[^(Edge)]/i.test(navigator.userAgent)
```